integration-test/src/main/java/com/sequenceiq/it/cloudbreak/testcase/e2e/environment/EnvironmentStopStartTests.java

```diff
@@ -34,6 +35,9 @@
     @Inject
     private CredentialTestClient credentialTestClient;
 
+    @Inject
```

Comment:
I don't see this used

Reply:
I checked all imports and they are used. May I ask you to double-check?